homeassistant/components/recorder/__init__.py

```diff
@@ -138,13 +146,18 @@ def run_information(hass, point_in_time: Optional[datetime] = None):
     if not db_url:
         db_url = DEFAULT_URL.format(hass_config_path=hass.config.path(DEFAULT_DB_FILE))
 
+    db_max_retries = conf.get(CONF_DB_MAX_RETRIES)
```

Comment:
Please use dict[key] here, e.g.
```suggestion
    db_max_retries = conf[CONF_DB_MAX_RETRIES]
```

Reply:
I've updated __init__.py with the changes you suggested in d11d446a7b42f8c21c8fe09329caa1b488109a12